nvim/apiimp.go

```diff
@@ -123,6 +123,50 @@ func (b *Batch) BufferLines(buffer Buffer, start int, end int, strict bool, resu
 	b.call("nvim_buf_get_lines", result, buffer, start, end, strict)
 }
 
+// AttachBuffer activate updates from this buffer to the current channel.
```

Comment:
I recently updated the Nvim docs for the attach methods. https://neovim.io/doc/user/api.html#nvim_buf_attach()

```suggestion
// Activates buffer-update events on the channel.
```

Reply:
Also will fix.